composer/datasets/ade20k.py

```diff
@@ -474,42 +583,20 @@ def __init__(self,
                          batch_size=batch_size)
 
         # Define custom transforms
-        if split == 'train':
-            self.both_transform = torch.nn.Sequential(
-                RandomResizePair(min_scale=min_resize_scale,
-                                 max_scale=max_resize_scale,
-                                 base_size=(base_size, base_size)),
-                RandomCropPair(
-                    crop_size=(final_size, final_size),
-                    class_max_percent=0.75,
-                    num_retry=10,
-                ),
-                RandomHFlipPair(),
-            )
-
-            # Photometric distoration values come from mmsegmentation:
-            # https://github.com/open-mmlab/mmsegmentation/blob/aa50358c71fe9c4cccdd2abe42433bdf702e757b/mmseg/datasets/pipelines/transforms.py#L861
-            r_mean, g_mean, b_mean = IMAGENET_CHANNEL_MEAN
-            self.image_transform = torch.nn.Sequential(
-                PhotometricDistoration(brightness=32. / 255, contrast=0.5, saturation=0.5, hue=18. / 255),
-                PadToSize(size=(final_size, final_size), fill=(int(r_mean), int(g_mean), int(b_mean))))
-
-            self.annotation_transform = PadToSize(size=(final_size, final_size), fill=0)
-        else:
-            self.both_transform = None
-            self.image_transform = transforms.Resize(size=(final_size, final_size),
-                                                     interpolation=TF.InterpolationMode.BILINEAR)
-            self.annotation_transform = transforms.Resize(size=(final_size, final_size),
-                                                          interpolation=TF.InterpolationMode.NEAREST)
+        self.transforms = build_ade20k_transformations(split=split,
```

Comment:
I feel having a separate return tuple name `self.both_transforms, self.image_transforms, self.target_transforms = build_ade20k_transformations(split=split,(...)` makes more sense since it is easy to understand and read here and during `def __getitem__()` method. wdyt ?

Reply:
I was conflicted if I should make it more verbose, but the clarity is worth the extra space, I'll add it :)